composer.json

```diff
@@ -82,7 +81,8 @@
         "white-october/pagerfanta-bundle": "^1.0",
         "php-amqplib/rabbitmq-bundle": "^1.8",
         "predis/predis": "^1.0",
-        "javibravo/simpleue": "^1.0"
+        "javibravo/simpleue": "^1.0",
+        "tecnickcom/tc-lib-pdf": "dev-master"
```

Comment:
Wow, might be a good idea to keep around before merging it right now. Why not targeting 2.2 branch? So we'll be able to give more tests on it